ee/modules/110-istio/openapi/doc-ru-config-values.yaml

```
@@ -71,6 +71,8 @@ properties:
     properties:
       enabled:
         description: Включить отправку трейсов в коллектор и интегрировать их отображение в kiali.
+      sampling:
+        description: Значение частоты дискретизации в процентах. Частота дискретизации должна находиться в диапазоне от 0.0 до 100.0 с точностью до 0.01.
```

```suggestion
        description: |
          Значение процентной доли запросов, которые будут сэмплированы для отправки в jaeger. Значение должно находиться в диапазоне от 0.0 до 100.0 с точностью до 0.01. Значение можно переопределить для каждого пода индивидуально при помощи аннотации:
          ```
          proxy.istio.io/config: |
            tracing:
              sampling: 100.0
          ```
```